app/js/components/layouts/ShareButton.react.js

```diff
@@ -1,31 +1,31 @@
-import React from 'react'
+import React from 'react';
+import $ from 'jquery';
 
 class SharePopup extends React.Component {
-  constructor(props, context){
-    super(props)
+  constructor(props) {
+    super(props);
+  }
 
-   }
-
-   componentDidMount(){
-     $(".share-popup").popup();
-     new Clipboard('.share-popup');
-     $(".share-popup").click(function(){
+   componentDidMount() {
+    //  $('.share-popup').popup();
+    //  new Clipboard('.share-popup');
+     $('.share-popup').click(function() {
        window.getSelection().removeAllRanges();
-     })
+     });
    }
 
-   render () {
-     let dom_id = this.props.dom_id;
-     let text_to_copy = this.props.text_to_copy;
-     let custom_class = this.props.custom_class;
+   render() {
+     let domId = this.props.dom_id;
```

Comment:
Part of the gradual change to camelcase. Plus `text_copy, custom_class` were deleted.